### PR TITLE
Newsletter-138:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**
After #521 is merged, the link(`[news137 taproot activation]`) can be updated.

### CODE REVIEW

File: _posts/ja/newsletters/2021-03-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    - *<!--lying-->偽り:* オラクルが、ユーザーが間違いだと分かっている結果に署名する不正。
+      これはほとんどの場合、ユーザーのコントラクトソフトウェアが利用できない証拠に依存するため、
+      この種の不正の証明は、元のコントラクトとオラクルが署名した結果を比較できるユーザーが、
+      手動で検証する必要があります。
+
+    議論の参加者は全員、v0の仕様に対して手間がかかりすぎるのではないかという懸念がありましたが、
+    多義性に対する証明の提供に賛成しているようでした。間を取る解決策として、
+    偽りに対する不正の証明へ焦点をあてる提案がされました。
+    これらの証明の形式が確立されると、ソフトウェアを更新して、
+    同じオラクルとイベントに対して多義性の証明を作成できるようになります。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-03-newsletter.md:45]:
```suggestion
    同じオラクルとイベントに対する2つの偽りに対する不正の証明を利用して、多義性の証明を作成できるようになります。
```

File: _posts/ja/newsletters/2021-03-03-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    - *<!--equivocation-->多義性:* オラクルが同じイベントに複数回署名し、矛盾する結果を生成する不正。
+      この不正の証明は、第三者を信用することなく、ソフトウェアによって自動的に検証できます。
+
+    - *<!--lying-->偽り:* オラクルが、ユーザーが間違いだと分かっている結果に署名する不正。
+      これはほとんどの場合、ユーザーのコントラクトソフトウェアが利用できない証拠に依存するため、
+      この種の不正の証明は、元のコントラクトとオラクルが署名した結果を比較できるユーザーが、
+      手動で検証する必要があります。
+
+    議論の参加者は全員、v0の仕様に対して手間がかかりすぎるのではないかという懸念がありましたが、
+    多義性に対する証明の提供に賛成しているようでした。間を取る解決策として、
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-03-03-newsletter.md:42]: I think this 'intermediate' means 'middle step of the solution' .

```suggestion
    多義性に対する証明の提供に賛成しているようでした。中間段階の解決策として、
```

File: _posts/ja/newsletters/2021-03-03-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+[hwi old process]: https://github.com/bitcoin-core/HWI/blob/7b34fc72c5b2c5af216d8b8d5cd2d2c92b6d2457/docs/examples/bitcoin-core-usage.rst
+[hwi release]: /en/newsletters/2019/05/07/#basic-hardware-signer-support-through-independent-tool
+[hwi new process]: https://github.com/bitcoin/bitcoin/blob/master/doc/external-signer.md
+[signer gui]: https://github.com/bitcoin-core/gui/pull/4
+[hwi rtd]: https://hwi.readthedocs.io/en/latest/?badge=latest
+[hwi examples]: https://hwi.readthedocs.io/en/latest/examples/index.html
+[hwi policy]: https://hwi.readthedocs.io/en/latest/devices/index.html#support-policy
+[X.509 certificates]: https://en.wikipedia.org/wiki/X.509
+[sighash types]: https://btcinformation.org/en/developer-guide#signature-hash-types
+[news137 taproot activation]: /en/newsletters/2021/02/24/#taproot-activation-discussion
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/ja/newsletters/2021-03-03-newsletter.md:132]: https://github.com/bitcoinops/bitcoinops.github.io/pull/521 is merged now @AkioNak

**Comment by azuchi** [comment]
@AkioNak Thanks! I fixed.

**Comment by AkioNak** [comment]
@azuchi Thanks. It looks good to me.

**Review by bitschmidty — APPROVED** [review]
ACK 606948578b8845b7f7ba7ec8682fa5cac550d34d

**Comment by bitschmidty** [comment]
Thanks @azuchi and @AkioNak ! ❤️

**Pull request merged**